app/components/SidebarLayout.tsx

```diff
@@ -116,6 +118,20 @@ export default function SidebarLayout({ children }: { children: ReactNode }) {
                 label="Locations"
                 href="/locations"
               />
+              <SideBarListItem
+                isCollapsed={isCollapsed}
+                isCurrentPage={segments.includes("fatalities")}
+                Icon={FaHeart}
+                label="Fatalities"
+                href="/fatalities"
+              />
+              <SideBarListItem
+                isCollapsed={isCollapsed}
+                isCurrentPage={segments.includes("create-crash-record")}
+                Icon={FaFileCirclePlus}
+                label="Create crash"
+                href="/create-crash-record"
+              />
```

Comment:
This `Create crash` sidebar item is an [upstream change](https://github.com/cityofaustin/vision-zero/pull/1637/files#r1929148700)